.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,7 +2,7 @@
 name: Bug report
 about: Create a report to help us improve
 title: ''
-labels: "Type: Bug"
+labels: "Type: Bug, State: Backlogged, Needs: Triage"
```

Comment:
This is all a single string in yaml; needs to be like so:

```yaml
labels: ["Type: Bug", "State: Backlogged", "Needs: Triage"]
```

(I usually use https://www.google.com/search?q=online+yaml+to+json to test yaml syntax :P )

Ditto elsewhere

Reply:
@cdrini I made a rookie mistake and polluted all the commits since I opened this into this branch. For your convinece, here are the commits that relevant to your last comment: 4234fad5d59b95537981e78222c44125b837685c , 0a7648b6f9ca0d675872d73067362b4879736218 , 0b1da3ebeac8e23c286377fa272bd65a93b99222

I'll make a new PR once the above commits are approved